docs/source/_static/theme_overrides.css

```diff
@@ -25,14 +25,38 @@
   --pst-font-weight-heading: 600;
 }
 
-/* Change header hight to make the logo a bit larger */
+/* Change header height to make the logo a bit larger */
 /* only on wider screens */
-@media only screen and (min-width: 1170px){
+
+@media only screen and (min-width: 1200px){
```

Comment:
This matches the width at which the PyData Sphinx theme hides the secondary (right) sidebar: https://github.com/pydata/pydata-sphinx-theme/blob/d76892d437e3c0dc8a059741a84afbd6cc458509/src/pydata_sphinx_theme/assets/styles/variables/_bootstrap.scss#L14

Setting this to that same width reduces the number of jumpy layout changes that happen when you're resizing the screen.